src/SixLabors.Shapes.Text/SixLabors.Shapes.Text.csproj

```diff
@@ -31,6 +31,10 @@
     <Compile Include="..\Shared\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="..\..\standards\**\*.cs" />
```

Comment:
We should introduce the breaking change properly and use  `shared-infrastructure`